c/meterpreter/source/extensions/stdapi/server/stdapi.c

```diff
@@ -55,6 +55,7 @@ Command customCommands[] =
 	COMMAND_REQ("stdapi_sys_process_close", request_sys_process_close),
 	COMMAND_REQ("stdapi_sys_process_execute", request_sys_process_execute),
 	COMMAND_REQ("stdapi_sys_process_kill", request_sys_process_kill),
+	COMMAND_REQ("stdapi_sys_process_nkill", request_sys_process_nkill),

```

Comment:
Rather than having a whole separate function that does this, is it not possible to merge this into the existing `request_sys_process_kill` function and check the TLV values that are passed in? This means that `kill` could take a combination of PIDs and process names and kill them all.